.github/workflows/main-workflow.yml

```diff
@@ -10,6 +10,8 @@ on:
 jobs:
   test:
     uses: ./.github/workflows/test-base.yml
+    secrets:
```

Comment:
This is the workflow when things are merged to main. Do we need to run codacy on those?

Reply:
Yes, it should run on main as well as branches.